ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -121,13 +125,25 @@ public NetworkMetrics(MetricRegistry registry) {
     selectorNodeMetricMap = new HashMap<String, SelectorNodeMetric>();
   }
 
-  public void initializeSelectorMetricsIfRequired(final AtomicLong activeConnections) {
+  /**
+   * Initializes a few network metrics for the selector
+   * @param activeConnections count of current active connections
+   * @param connectionsPendingHandshakeCount Count of connections that are awaiting handshake completion
+   */
+  public void initializeSelectorMetricsIfRequired(final AtomicLong activeConnections,
+      final AtomicLong connectionsPendingHandshakeCount) {
```

Comment:
again, could be consistent with `count` (or preferably `num`) for the two variables.